.github/workflows/build-and-test.yaml

```diff
@@ -51,7 +51,8 @@ jobs:
         run: make lint
 
       - name: Test
-        run: make test
+          run: make test
```

Comment:
Here is the reason. The yaml indentation is wrong 
```suggestion
        run: make test
```

Reply:
thx for your help